browser/components/newtab/RemoteAboutNewTab.jsm

```diff
@@ -343,6 +363,8 @@ let RemoteAboutNewTab = {
     PlacesProvider.links.on("clearHistory", this.placesClearHistory.bind(this));
     PlacesProvider.links.on("linkChanged", this.placesLinkChanged.bind(this));
     PlacesProvider.links.on("manyLinksChanged", this.placesManyLinksChanged.bind(this));
+    NewTabPrefsProvider.prefs.on("browser.newtabpage.enabled", this.setEnabled.bind(this));
+    NewTabPrefsProvider.prefs.on("browser.newtabpage.enhanced", this.setEnhanced.bind(this));
```

Comment:
we need to make one. We can send the message to the page, and write an issue on mozilla/remote-newtab to handle it